src/Domain/LeanCode.CQRS.RemoteHttp.Server/BaseRemoteObjectHandler.cs

```diff
@@ -68,14 +68,25 @@ public async Task<ExecutionResult> ExecuteAsync(HttpContext context)
             catch (UnauthenticatedException)
             {
                 result = ExecutionResult.Fail(StatusCodes.Status401Unauthorized);
+
+                Logger.Warning(
+                    "Unauthenticated user requested {@Object} of type {Type}, which requires authorization",
+                    obj,
+                    type);
             }
-            catch (InsufficientPermissionException)
+            catch (InsufficientPermissionException ex)
             {
                 result = ExecutionResult.Fail(StatusCodes.Status403Forbidden);
+
+                Logger.Warning(
+                    "Authorizer {Authorizer} failed to authorize the user to run {@Object} of type {Type}",
```

Comment:
Why don't you pass ex as first parameter?

Reply:
There's nothing of value in the exception itself.